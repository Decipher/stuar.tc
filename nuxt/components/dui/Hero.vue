<template>
  <div class="hero min-h-screen bg-base-200">
    <div class="container mx-auto">
      <div class="flex-col hero-content md:flex-row-reverse">
        <slot name="image" />
        <div>
          <slot />
        </div>
      </div>
    </div>
  </div>
</template>
