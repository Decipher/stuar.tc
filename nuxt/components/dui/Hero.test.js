import { mount } from '@vue/test-utils'
import Hero from './Hero.vue'

describe('Hero', () => {
  test('is a Vue instance', () => {
    const wrapper = mount(Hero)
    expect(wrapper.vm).toBeTruthy()

    // Expect HTML to match snapshot.
    expect(wrapper.html()).toMatchSnapshot()
  })
})
