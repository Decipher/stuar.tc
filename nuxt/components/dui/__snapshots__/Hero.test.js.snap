// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero is a Vue instance 1`] = `
"<div class=\\"hero min-h-screen bg-base-200 py-40\\">
  <div class=\\"container mx-auto\\">
    <div class=\\"flex-col hero-content md:flex-row-reverse\\">
      <div class=\\"flex-1\\"></div>
    </div>
  </div>
</div>"
`;
